### DISCUSSION TITLE
Menu Configuration

### DISCUSSION THREAD

**Discussion by rafizan-baharum** [ShankyTiwari/ng-material-multilevel-menu#215] [General]
Is there any configuration to make parent menu uncollapsed. I need to be able to show all the items, all the time. I've tried collapseOnSelect:false and also forcing expand with menuService.setMenuExapandCollpaseStatus.

**Comment by ShankyTiwari** [comment]
> Is there any configuration to make the parent menu uncollapsed
I didn't catch you @rafizan-baharum!

Normally Parent Node is uncollapsed when you have selected one of its child Node as shown below,
![image](https://github.com/ShankyTiwari/ng-material-multilevel-menu/assets/15548024/652a9913-88e7-4053-8031-7f5eb0be19c0)

Are you talking about uncollapsing the Sibling Node?

Thanks

P.S. Sorry for the late reply.
